### PR TITLE
fix(issue#29): added new CS questions

### PR DESCRIPTION
**Pull request by a-cole55**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by a-cole55** [comment]
two new CS questions added to full-quiz.ts file

**Comment by jdwilkin4** [comment]
HI @a-cole55 ! 

It looks like you have some merge conflicts and your branch is 12 commits ahead and 1 commit behind the fcc main branch. 
I would read through this github issue to resolve the issue
https://stackoverflow.com/questions/41283955/github-keeps-saying-this-branch-is-x-commits-ahead-y-commits-behind

Hope that helps!

File: src/data/full-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,26 @@
 const fullQuiz = [
+  {
+    Question:
+      "What is the basic language of a computer?",
+    Answer: "Binary",
+    Distractor1: "HyperText Markup Language",
+    Distractor2: "Cascading Style Sheets",
+    Distractor3: "JavaScript",
+    Explanation:
+      "Computers are simplistic and understand binary, other languages (such as Python, Java, etc) were created to help humans write instructions for computers and interpret it.",
```
Comment (jdwilkin4) [inline src/data/full-quiz.ts:10]: I think the explanation should be two sentences. 

Proposed solution: 
Computers are simplistic and understand binary. Programming languages such as Python and Java were created to help humans write instructions for computers and interpret it.

File: src/data/full-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,26 @@
...
+    Link: "https://youtu.be/F0WoVEr0-44?t=336",
+  },
+  {
+    Question:
+      "Where is the CPU located in a computer?",
```
Comment (jdwilkin4) [inline src/data/full-quiz.ts:15]: This question already exists in the database. 
Please remove it. 👍

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Left a few comments 👍

**Pull request merged**